content/realtime/channel-params.textile

```diff
@@ -139,3 +139,111 @@ h3(#rewind-examples-mqtt). Rewind example with MQTT
     ...
   });
 ```
+h2(#delta). Delta parameter
+
+The @delta@ channel parameter expresses the intent to receive the delta (diff) between the previous and current message instead of the current message in full on a given channel. This parameter is oblivious to the sender and only affects the subscriber using it. I.e. the sender sends messages in full and Ably Realtime would compute and send the message deltas to any subscriber using this parameter. 
```

Comment:
> Ably Realtime would compute

Just 'Ably' -- referring to ourselves "Ably Realtime" sounds quite formal outside of legal documents.

(There's places where we refer to things like "Ably Realtime client libraries", but that's because those are referring to "{Ably} {Realtime client libraries}", as opposed to our REST client libraries)

Reply:
fbf72fa